tests/Fixer/ClassNotation/OrderedTypesFixerTest.php

```diff
@@ -40,7 +40,7 @@ public function testFix(string $expected, ?string $input = null, ?array $config
     }
 
     /**
-     * @return iterable<string, array{string, 1?: ?string, 2?: array<string, mixed>}>
```

Comment:
why removing the string? it is a good detail that I would love to have for all data provider - it stats for test case description (`yield DESCRIPTION => CASE`)